addons/controllers/clusterbootstrap_controller.go

```diff
@@ -210,6 +213,24 @@ func (r *ClusterBootstrapReconciler) reconcileNormal(cluster *clusterapiv1beta1.
 		return ctrl.Result{}, err
 	}
 
+	_, err = r.createOrPatchResourcesForCorePackages(cluster, clusterBootstrap, remoteClient, log)
```

Comment:
Kapp controller is also a core package, could you try include `createOrPatchKappPackageInstall ` in this function as well?

Reply:
The requeAfter values are different for kappcontroller than for CorePackages. 
Not sure we can lump them together, but if we can make the same requeAdfter for kappcontroller it can be done. 
I'll submit on its own commit so we can roll out if we don't like it

Reply:
It's non-blocking, I guess what I meant was to put `createOrPatchKappPackageInstall` inside `createOrPatchResourcesForCorePackages`, as the scope of latter seems include the first function.

Reply:
yes. but in the original logic if createOrPatchKappPackageInstall fails reconcileNormal returns ctrl.Result{}, err, whilst if the failure is when installing a corepackage (createOrPatchResroucesForCorePackages) then reconcileNormal returns  ctrl.Result{requeueAfter: constants.RequeueAfterDuration}, err 

If we merge the functions, then reconcileNormal will have to return ctrl.Result{requeueAfter: constants.RequeueAfterDuration}, err 
for both. (maybe that is what we want, but probably best done in a different pr then). 
Not sure what the effects might be. I think we best not change it, we would only be moving the code around for neatness, I would rather not, just in case it could introduce a bug.

Reply:
It's non-blocking. Let's defer this one.